packages/react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@farfetched/react",
-  "version": "0.2.1",
+  "version": "0.3.0",
```

Comment:
Repository generates versions automatically, check README.md for more detailed development flow description. So, please revert this change.